html/form/advanced.phtml

```diff
@@ -1,7 +1,7 @@
 <form id="smly" class="container" action="<?php echo SS_PLUGIN_URL; ?>/subscribe.php" method="post">
 	<p class="error" style="padding:15px;background-color:#f2dede;margin:0 0 10px;display:none"></p>
 	<p class="success" style="padding:15px;background-color:#dff0d8;margin:0 0 10px;display:none"><?php echo esc_html__( 'Thank you for subscribing to our newsletter.', 'wp_sendsmaily' ); ?></p>
-
+	<input type="hidden" name="lang" value="<?php echo ( defined( 'ICL_LANGUAGE_CODE' ) ) ? ICL_LANGUAGE_CODE : 'en'; ?>" />
```

Comment:
Defaulting to English might not be the best approach. User's site can be in single language and that language might not be English.

I would exclude `lang` field entirely if `ICL_LANGUAGE_CODE` is not defined. Or to keep the previous behaviour and default `lang` to empty value.